### DISCUSSION TITLE
Rack 3 Release

### DISCUSSION THREAD

**Discussion by tenderlove** [rack/rack#1854] [General]
I just wanted to start a discussion with regard to releasing Rack 3.  I will ship the gem, but I want to get blockers etc tracked for the release.  I've started a project [here](https://github.com/rack/rack/projects/2).  Please add any issues / pull requests to the project so that we can track progress towards release.  Once we are ready, I will ship the gem.

Thanks.

**Comment by ioquatix** [comment]
Thanks for organising this.
